### PR TITLE
Hacky fix for byond bug breaking menu preferences

### LINKED ISSUE #27281: Icon scaling mode preference never saves
Even if you change to something other than nearest neighbor, it will reset.

@MrStonedOne just in case you forgot!

### PR DESCRIPTION
**Pull request by MrsTonedOne**
:cl: 
fix: Fixed bug breaking menu preferences around icons
/:cl:

Technically both were broke, but byond already remembers icon size across loads.

This fixes both.

Web editor edit so i still have to download this and test it out. 

fixes #27281

### CODE REVIEW

File: interface/menu.dm [hunk 1]
```diff
@@ -76,6 +76,23 @@ GLOBAL_LIST_EMPTY(menulist)
 /datum/verbs/menu/Icon/Size
 	checkbox = CHECKBOX_GROUP
 	default = /datum/verbs/menu/Icon/Size/verb/iconstretchtofit
+/datum/verbs/menu/Icon/Size/Load_checked(client/C)
+	var/atom/verb/verbpath = Get_checked(C)
```
Comment (RemieRichards) [inline interface/menu.dm]: don't cast these, they're paths not atoms. (and verb doesn't inherit from atom anyway so thats super weird)

Comment (MrStonedOne) [inline interface/menu.dm]: byond acts odd if you don't. it doesn't matter for this use case, but if you want to access `verbpath.name` from a verb path, it has to be casted as a verb.

verb path and verb object are one in the same.

(or would you rather i do `verbpath:name` =P)

File: interface/menu.dm [hunk 2]
```diff
@@ -76,6 +76,23 @@ GLOBAL_LIST_EMPTY(menulist)
...
+	if (!verbpath || !(verbpath in typesof("[type]/verb")))
```
Comment (nicbn) [inline interface/menu.dm]: Don't add spaces after if, switch, else etc. on SS13 code. This line should be, for example:
`if(!verbpath || !(verbpath in typesof("[type]/verb")))`

Comment (RemieRichards) [inline interface/menu.dm]: Nope, we don't enforce that.
It's specifically in our "not enforced" section as MrStonedOne likes to do it.

Comment (MrStonedOne) [inline interface/menu.dm]: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#enforced-not-enforced

Comment (lzimann) [inline interface/menu.dm]: [2nd item](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#enforced-not-enforced)

**Comment by MrStonedOne** [comment]
real fix found

File: interface/menu.dm (hunk 2 above)
Comment (nicbn) [inline interface/menu.dm]: Oh my, I was doing ugly coding thinking it was necessary or some kind of convention

Comment (MrStonedOne) [inline interface/menu.dm]: Ya, read the [specs](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#specification), it kinda goes over dos and dont's and in some case, why.

**Pull request closed without merging**